### PR TITLE
Host yards

### LINKED ISSUE #33: Get yards by Host ID
Get /api/v1/host/host_id/yards

### LINKED ISSUE #39: Get Host Yards
/api/v1/hosts/host_id/yards

### PR DESCRIPTION
**Pull request by jbranham1**
Created host yards endpoint

Close #39 
Close #33

### CODE REVIEW

**Review by abreaux26 — APPROVED** [review]
💪🏼 💪🏼 💪🏼 💪🏼 💪🏼 💪🏼 💪🏼 💪🏼

File: app/controllers/api/v1/yards_controller.rb [hunk 1]
```diff
@@ -1,5 +1,11 @@
 class Api::V1::YardsController < ApplicationController
-  before_action :validate_params, only: :show
+  before_action :validate_host_params, only: [:index]
+  before_action :validate_yard_params, only: [:show]
+
```
Comment (amsmyth1) [inline app/controllers/api/v1/yards_controller.rb:4]: I would love to know what these are doing for my own understanding

File: config/routes.rb [hunk 2]
```diff
@@ -5,6 +5,9 @@
     namespace :v1 do
       resources :purposes, only: [:index]
       resources :yards, except: [:index]
+      resources :hosts, only: [] do
+        resources :yards, only: [:index]
```
Comment (amsmyth1) [inline config/routes.rb:9]: I've never seen this done with an empty array, so smart!

**Review by amsmyth1 — APPROVED** [review]
Looks great to me! These tests are tight

**Comment by amsmyth1** [comment]
Pulled it down to my local, everything is operating as expected. Great job team!

**Pull request merged**